civihr_default_theme/scss/pages/_report-settings-age-group.scss

```diff
@@ -0,0 +1,3 @@
+table.civihr-employee-portal-civihr-report-settings-age-group {
```

Comment:
The name of the file should be `_reports.scss`, and this file is only meant to contain all the overrides that you do of a generic component for the reports pages.